cli/tests/pcluster/validators/test_directory_service_validators.py

```diff
@@ -111,6 +113,49 @@ def test_ldap_tls_reqcert_validator(ldap_tls_reqcert, expected_message):
     assert_failure_messages(actual_failures, expected_message)
 
 
+@pytest.mark.parametrize(
+    "password_secret_arn, error_from_secrets_manager, expected_message, expected_error",
```

Comment:
just secret_arn to be more generic

Reply:
This fix is conditioned to the generalization of `PasswordSecretArnValidator` to `SecretValidator`, which at this point does not really convince me.

Reply:
I think this could be modified in any case, anyway not strictly needed for now.